evap/evaluation/templates/base.html

```diff
@@ -211,5 +211,18 @@
         </script>
 
         {% block additional_javascript %}{% endblock %}
+
+        <script type="text/javascript"> 
+            document.querySelectorAll("[data-earliest-legal-date]").forEach(element => {
```

Comment:
I would refrain from "legal" again, how about something like `data-minimum-date` and `data-minimum-date-reason`?